Add lensiro.json for domain configuration

social-lensiro opened the pull request:
add lensiro.is-a.dev

<!--
YOU MUST FILL OUT THIS TEMPLATE FOR YOUR PR TO BE ACCEPTED!
-->

# Requirements
<!-- Your domain MUST pass ALL the requirements below, otherwise it WILL BE DENIED. -->
<!-- Change each checkbox to [x] (all lowercase, with no spaces between the brackets) to mark it as completed. -->

- [x] I **agree** to the [Terms of Service](https://is-a.dev/terms). <!-- Your request MUST follow the TOS to be approved. -->
- [x] My file is following the [domain structure](https://docs.is-a.dev/domain-structure/). <!-- Your JSON file is in the domains directory, the name is valid, etc. -->
- [x] My website is **reachable** and **completed**. <!-- We do not permit simple "Hello, world!" or simply copied template websites with minimal changes. -->
- [x] My website is **software development** related. <!-- Only your root subdomain needs to meet this requirement. -->
- [x] My website is **not for commercial use**. <!-- Your website's purpose should not be to generate any form of revenue or profit. -->
- [x] I have provided contact information in the `owner` key. <!-- Provide your email in the `email` field or another platform (e.g. X/Twitter or Discord) for contact. -->
- [x] I have provided a preview of my website below. <!-- This step is required for your PR to be approved. -->

# Website Preview
<!-- Provide a link AND screenshot of your website below. -->

social-lensiro committed:
Add lensiro.json for domain configuration

add lensiro.is-a.dev

STICKnoLOGIC commented on lensiro.json:
```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "lensiro.is-a.dev",
```
```suggestion
```

STICKnoLOGIC reviewed (changes requested):
provide website preview (link)
resolve issues below

STICKnoLOGIC commented on lensiro.json:
move this file inside of domains folder

social-lensiro committed:
Add GitHub handle to lensiro.json owner

fix domain config: add github and rename to records

social-lensiro commented:
✅ Fixed all requested changes:
- Moved file to /domains folder
- Added GitHub username
- Changed "record" to "records"
Preview: https://publish.obsidian.md/manual-lensiro

STICKnoLOGIC reviewed (changes requested):
<img width="1599" height="527" alt="Image" src="https://github.com/user-attachments/assets/e3605502-739c-47be-a70d-b15f2efdc5f1" />

- the provided website not found (stop using AI bro to reply this PR)
- resolve issues below

social-lensiro committed:
Update lensiro.json

fix: remove domain field, update JSON structure
 sorry for use ai :)

social-lensiro committed:
Merge branch 'main' into main

social-lensiro commented:
<img width="2560" height="1379" alt="image" src="https://github.com/user-attachments/assets/fcb6ae7b-9565-439d-bd85-cba30716a5f4" />

STICKnoLOGIC reviewed (commented):
closing:
not software dev related, we only accept dev related portfolio site in our main subdomain
